pkg/cri/config/config.go

```diff
@@ -302,6 +302,13 @@ type PluginConfig struct {
 	// and if it is not overwritten by PodSandboxConfig
 	// Note that currently default is set to disabled but target change it in future together with EnableUnprivilegedPorts
 	EnableUnprivilegedICMP bool `toml:"enable_unprivileged_icmp" json:"enableUnprivilegedICMP"`
+	// EnableCDI indicates to enable injection of the Container Device Interface Specifications
+	// into the OCI config
+	// For more details about CDI and the syntax of CDI Spec files please refer to
+	// https://github.com/container-orchestrated-devices/container-device-interface.
+	EnableCDI bool `toml:"enable_cdi" json:"enableCDI"`
+	// CDISpecDirs is the list of directories to scan for Container Device Interface Specifications
+	CDISpecDirs []string `toml:"cdi_spec_dirs" json:"cdiSpecDirs"`
```

Comment:
need to update the cri config docs...
over here: https://github.com/containerd/containerd/blame/main/docs/cri/config.md#L122

Reply:
please make mention of the defaults for the admins...
EnableCDI:   false,
CDISpecDirs: []string{"/etc/cdi", "/var/run/cdi"},

Reply:
after merge here.. we could push to the readme over in container-orchestrated-devices/container-device-interface to mention the required change to cri config here... to run the demo/device integration..

Reply:
@mikebrow thank you for the reveiw! I've updated config.md as you've suggested and [created an issue](https://github.com/container-orchestrated-devices/container-device-interface/issues/59) in the COD/CDI github project to prepare admin docs. Let me know if anything else is needed to lgtm this PR.